.github/DISCUSSION_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,32 @@
+title: "[Feature] "
+labels: ["Ideas"]
+body:
+  - type: checkboxes
+    attributes:
+      label: I've already checked other feature proposals
+      required: true
+  - type: dropdown
+    attributes:
+      label: Is your feature request related to a problem?
+      options:
+        - Yes
+        - No
```

Comment:
I don't understand this one. I would get really confused about what a "yes" and a "no" actually means here. Maybe just remove it?